peer/core.yaml

```diff
@@ -97,11 +97,14 @@ peer:
 
     # The Address this Peer will listen on
     listenAddress: 0.0.0.0:30303
-    # The Address this Peer will bind to for providing services
+
+    # The Address advertised to other services that will connect to this Peer
+    # Chaincode deployment will use this to create chaincode docker containers
+    # (This is NOT a binding address for the listening port)
     address: 0.0.0.0:30303
```

Comment:
I like the more detailed info on those parameters but it is still clear for me what address is used for. I mean if it is listening on listenAddress how can other services connect to it on address? Would it make sense to keep listenAddress but change address to something more descriptive? E.g. xyzAddress (where xyz is something useful of course :))